backend/api/tests/test_admin_views.py

```
@@ -406,4 +406,130 @@ def test_promote_unauthorized(self, user_client, regular_user):
+        assert all(rec['upload_status'] == 'completed' for rec in response.data)
+        assert any(rec['filename'] == 'test_recording.webm' for rec in response.data)
+
+
+        # Filter by pending status
+        response = admin_client.get(url, {'status': 'pending'})
+        assert response.status_code == status.HTTP_200_OK
+        assert len(response.data) == 1
+        assert response.data[0]['filename'] == 'pending_recording.webm'
+        assert response.data[0]['upload_status'] == 'pending'
+
+    def test_unauthorized_access(self, user_client, test_webcam_recording):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_unauthorized_access`

Add return type annotation: `None`

(ANN201)